digdag-storage-s3/src/main/java/io/digdag/storage/s3/S3StorageFactory.java

```diff
@@ -30,6 +31,13 @@ public Storage newStorage(Config config)
             client.setEndpoint(config.get("endpoint", String.class));
         }
 
+        if (config.has("path-style-access")) {
```

Comment:
this should be `config.get("path-style-access", boolean.class, false)` otherwise `archive.s3.path-style-access = false` enables path-style-access unexpectedly.